specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -3951,6 +4185,13 @@
           "$ref": "#/definitions/VirtualHubRouteTable",
           "description": "The routeTable associated with this virtual hub."
         },
+        "routeTables": {
+          "type": "array",
+          "description": "List of all RouteTables associated with this VirtualHub.",
+          "items": {
+            "$ref": "#/definitions/HubRouteTable"
+          }
+        },
```

Comment:
this is not an acceptable pattern anymore.  It breaks RBAC allowing users to view/edit child resources inside the parent.  They should only be exposed as a child resource OR as an array inside the parent, not both

Reply:
Done